product-scenarios/scenarios-common/src/main/java/org/wso2/am/scenario/test/common/ScenarioTestBase.java

```diff
@@ -275,32 +275,36 @@ public void deleteRole(String role, String adminUsername, String adminPassword)
         }
     }
 
-    public void isAPIVisibleInStoreForAnonymousUser(String apiName, String tenantDomain)
+    public void isAPIVisibleInStore(String apiName, APIStoreRestClient apiStoreRestClient)
             throws APIManagerIntegrationTestException {
-        APIStoreRestClient apiStoreClient = new APIStoreRestClient(storeURL);
-
-        long waitTime = System.currentTimeMillis() + 5 * 1000;
+        long waitTime = System.currentTimeMillis() + ScenarioTestConstants.WAIT_TIME;
```

Comment:
shall we use a bit specific name for this: suggestion: API_APPEAR_IN_STORE_TIMEOUT_MILLIS

Reply:
Ack. will send the change with another PR